cmd/peggo/keys.go

```diff
@@ -390,8 +401,9 @@ func keyringForPrivKey(name string, privKey sdkcryptotypes.PrivKey) (keyring.Key
 	}
 
 	armored := sdkcrypto.EncryptArmorPrivKey(privKey, tmpPhrase, privKey.Type())
+	encodingConfig := umeeapp.MakeEncodingConfig()
```

Comment:
Maybe we already have a codec somewhere? Creating a whole app code for keyring is an overkill.

Reply:
I will check that

Reply:
done at https://github.com/umee-network/peggo/pull/343/commits/5de35e1e8f9ddd0648962e67e43856beb048029f